web/src/components/apps/AppDetailPage.jsx

```diff
@@ -378,6 +378,7 @@ class AppDetailPage extends Component {
                         makingCurrentRelease={this.state.makingCurrentRelease}
                         redeployVersion={this.redeployVersion}
                         redeployVersionErrMsg={this.state.redeployVersionErrMsg}
+                        adminConsoleMetadata={this.props.adminConsoleMetadata}
```

Comment:
not sure if `adminConsoleMetadata` is being passed from Root.jsx to AppDetailPage.jsx as part of `{...props}` , so this might be undefined